### PR TITLE
fix config for notifications

### LINKED ISSUE #268: Slack notification for `trigger = "deploy"` fails
There is an issue when adding Slack notification for `trigger = "deploy"`.

Here is the TF resource:

```
resource "rollbar_notification" "deploy" {
  rule {
    filters {
      type      = "environment"
      operation = "eq"
      value     = "prod"
    }
    trigger = "deploy"
  }
  channel = "slack"
  config {
    channel = var.slack_channel
  }
}
```

As `show_message_buttons` is defined by default, terraform apply fails with:
```
│ Error: 1 Additional properties are not allowed (u'show_message_buttons' was unexpected)
│
│   with module.ds_prize_preference_notifications.rollbar_notification.deploy_prod,
│   on ../common/modules/rollbar-notifications/notifications_prod.tf line 1, in resource "rollbar_notification" "deploy_prod":
│    1: resource "rollbar_notification" "deploy_prod" {
```

### PR DESCRIPTION
**Pull request by pawelsz-rb**
## Description of the change

Fix config for notifications 

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Maintenance
- [ ] New release

## Related issues

- Fix #268

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers assigned
- [ ] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

File: rollbar/resource_notification_test.go [hunk 1]
```diff
@@ -165,6 +165,41 @@ func (s *AccSuite) TestNotificationUpdate() {
 	})
 }
 
+func (s *AccSuite) TestNotificationCreateSpecialEmail() {
+	notificationResourceName := "rollbar_notification.email_notification"
+	// language=hcl
+	config := `
+		resource "rollbar_notification" "email_notification" {
```
Comment (haggishunk) [inline rollbar/resource_notification_test.go:172]: Minor formatting question-- would it be OK to remove the leading whitespace on this line to have the `resource` block a bit more legible?

File: rollbar/resource_notification_test.go [hunk 2]
```diff
@@ -165,6 +165,41 @@ func (s *AccSuite) TestNotificationUpdate() {
...
+		CheckDestroy: nil,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					s.checkResourceStateSanity(notificationResourceName),
+					resource.TestCheckResourceAttr(notificationResourceName, "channel", "email"),
+					resource.TestCheckResourceAttr(notificationResourceName, "rule.0.trigger", "daily_summary"),
+					resource.TestCheckResourceAttr(notificationResourceName, "config.0.summary_time", "2"),
+					resource.TestCheckResourceAttr(notificationResourceName, "config.0.min_item_level", "critical"),
```
Comment (haggishunk) [inline rollbar/resource_notification_test.go:196]: Should we also include defs and tests for each of the email daily summary-related fields?

Comment (pawelsz-rb) [reply rollbar/resource_notification_test.go:197]: done

File: rollbar/resource_notification_test.go (hunk 1 above)
Comment (pawelsz-rb) [reply rollbar/resource_notification_test.go]: done, this is really a string, so we would need to do our formatting by hand. gofmt does not format multiple line strings really well.

**Review by haggishunk — APPROVED** [review]

**Pull request merged**